municipal_finance/resources.py

```diff
@@ -140,3 +146,54 @@ class Meta:
             'period_code',
             'item',
         ]
+
+
+class CapitalTypeV2Resource(resources.ModelResource):
+    class Meta:
+        model = CapitalTypeV2
+        import_id_fields = ['code']
+
+
+class CapitalItemsV1Resource(resources.ModelResource):
+    class Meta:
+        model = CapitalItemsV1
+        import_id_fields = ['code']
+
+
+class CapitalItemsV2Resource(resources.ModelResource):
+    class Meta:
+        model = CapitalItemsV2
+        import_id_fields = ['code']
+
+
+class CapitalFactsV1Resource(resources.ModelResource):
+    class Meta:
+        model = CapitalFactsV1
+        import_id_fields = [
+            'demarcation_code',
+            'period_code',
+            'item_code',
+            'function_code',
+        ]
+
+
+class CapitalFactsV2Resource(resources.ModelResource):
+    class Meta:
+        model = CapitalFactsV2
+        import_id_fields = [
+            'demarcation_code',
+            'period_code',
+            'item',
+            'function',
+            'capital_type',
+        ]
+
+
+class UIFWExpenditureFactsResource(resources.ModelResource):
```

Comment:
why do we have facts resources under production code? This makes me nervous

Reply:
@jbothma I've moved the resources for facts to the test folder where they are used.